api/db/repositories/customerRepository.js

```diff
@@ -7,19 +7,22 @@ exports.createCustomer = async (customerData) => {
 };
 
 exports.updateCustomer = async (customerId, updatedCustomerData) => {
-    return Customers.update(updatedCustomerData, 
+    return await Customers.update(updatedCustomerData, 
```

Comment:
remove async and await from Repositories if they only return one line. It gets awaited further on during the process like in domain or controller